scipy-notebook/Dockerfile

```diff
@@ -74,6 +74,10 @@ RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
 RUN ln -s $CONDA_DIR/envs/python2/bin/pip $CONDA_DIR/bin/pip2 && \
     ln -s $CONDA_DIR/bin/pip $CONDA_DIR/bin/pip3
 
+# Import matplotlib the first time to build the font cache.
+ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
+RUN $CONDA_DIR/envs/python2/bin/python -c "import matplotlib.pyplot"
```

Comment:
This qt4 issue is a transient problem created by anaconda. They released qt5 (the new default package for 'qt') but retained their matplotlib configuration that expects qt4 specifically. Seems like this was fixed upstream yesterday, so simply re-kicking the build might be enough. But `MPL_BACKEND=Agg` is safer anyway.